fmgr/interfaces.go

```diff
@@ -0,0 +1,30 @@
+package fmgr
+
+import (
+	"github.com/lightningnetwork/lnd/lnpeer"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// FundingUpdater is an interface implemented by some messages in BOLT 2 that
+// are allowed to update the funding state.
+type FundingUpdater interface {
+	// FundingChanID is a function that is used by messages that relate to some
+	// aspect of the funding flow. ChannelID in this context is not well-defined
+	// and may refer to the pending channel ID. This is really only used for the
+	// interface abstraction benefits.
+	FundingChanID() [32]byte
```

Comment:
is this used anywhere?

Reply:
No only to make some of the funding messages adhere to an interface. We can't do it with `lnwire.Error` since it has some custom logic with `isLinkUpdate` & `handleError`

Reply:
Its used in the peer